libs/go/aealite/protocols/message_test.go

```diff
@@ -39,7 +39,7 @@ func TestMessage(t *testing.T) {
 	message.Content = []byte(`{"performative": "request", "data": "hello"}`)
 
 	result := DialogueMessageWrapper{}
-	err := result.InitFromProtobuf(&message)
+	err := result.InitFromProtobufAndPerfofrmative(&message, "request")
```

Comment:
Typo! `InitFromProtobufAndPerfofrmative` > `InitFromProtobufAndPerformative`

Reply:
It is correct; I don't want to cause a conflict because of a typo. Will be addressed later